microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -87,31 +95,54 @@ For full support for the latest APIs in a release version of the SDK, the Runtim
+<!-- todo: "experimental" is missing from .NET API Ref:
+https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2webmessagereceivedeventargs.additionalobjects?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true
+Win32:
+https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalwebmessagereceivedeventargs?view=webview2-1.0.1724-prerelease&preserve-view=true#get_additionalobjects
+-->
 
-To help you evaluate the experimental APIs and share your feedback, use the [WebView2Feedback](https://github.com/MicrosoftEdge/WebViewFeedback) repo.
+To help you evaluate the Experimental APIs and share your feedback, use the [WebView2Feedback](https://github.com/MicrosoftEdge/WebViewFeedback) repo.
 
 
-### Moving from experimental APIs to stable APIs
+<!-- ------------------------------ -->
+#### Moving from Experimental APIs to Stable APIs
 
-Once an API has been moved from experimental to stable APIs, you need to move your app's code to the stable API.  Using experimental APIs is not recommended for production apps.  Follow these practices when moving your app from using experimental APIs to using stable APIs:
+Once an API has been moved from Experimental to Stable status, you need to move your app's code to the Stable API.  Using Experimental APIs or a Prerelease SDK is not recommended for production apps.  Follow these practices when moving your app from using Experimental APIs to using Stable APIs:
 
-*  In your project in Visual Studio, update your WebView2 SDK package version.  See [Install the WebView2 SDK](../how-to/machine-setup.md#install-the-webview2-sdk) in _Set up your Dev environment for WebView2_.
+*  In your project in Visual Studio, update your WebView2 SDK package version to a newer Prerelease SDK or Release SDK.  See [Install the WebView2 SDK](../how-to/machine-setup.md#install-the-webview2-sdk) in _Set up your Dev environment for WebView2_.
 
-*  Update your app's code to use stable APIs instead of experimental APIs (for COM).  The stable API will be supported with bug fixes, but the experimental API will be deprecated, and not available in the newer SDK.  After the release of an API as stable, the experimental version of that API is supported for two releases, in a deprecated state.  In subsequent versions of the SDK, experimental APIs might be modified, removed, or added.
+*  Update your app's code to use Stable APIs instead of Experimental APIs (for COM).  The Stable API will be supported with bug fixes, but the Experimental API will be deprecated, and not available in the newer (Prerelease or Release) SDK.  After an API is promoted to Stable, the Experimental version of that API is supported for two releases of the Prerelease SDK, in a deprecated state.  In subsequent versions of the Prerelease SDK, Experimental APIs might be modified, removed, or added.
```

Comment:
@novac42 @peiche-jessica -- I tried to address the question of whether we're meaning Prerelease or Release SDK.  This existing paragraph just talks about "the" SDK.  Pls review where I changed "SDK" to "Prerelease SDK" or "Release SDK".

Reply:
In subsequent versions of the SDK -- imo the SDK here means both the Prerelease and Release SDK. So another option is  just to keep the old 'the SDK' wording?

Reply:
The new wording conforms with "SDK here means both the Prerelease and Release SDK".

Against keeping the old "the SDK" wording: This versioning.md article is about the distinction between Prerelease vs. Release, so instead of saying "the SDK", in contexts such as this section, we should be specific which type/version is meant; that's a question the user in this context will have.

Reply:
@novac42 - About the existing sentence:
"In subsequent versions of the SDK, experimental APIs might be modified, removed, or added."
You wrote:
"In subsequent versions of the SDK -- imo the SDK here means both the Prerelease and Release SDK"

The sentence can only mean the Prerelease SDK, not the Release SDK, because Experimental APIs (while in Experimental status) are never in the Release SDK.  The context is: highly volatile; so, the sentence is discussing only the Prerelease SDK.